Actually, it looks like there is a test referencing the minion resource:
```
--- FAIL: TestResourceTuple (0.01s)
	builder_test.go:635: valid multiple with namespaced and non-namespaced types: unexpected error: no resource "minions" has been defined
```

--- a/pkg/api/register.go
+++ b/pkg/api/register.go
@@ -71,9 +71,6 @@ func init() {
 		&ThirdPartyResourceList{},
 		&ThirdPartyResourceData{},
 	)
-	// Legacy names are supported
-	Scheme.AddKnownTypeWithName("", "Minion", &Node{})
-	Scheme.AddKnownTypeWithName("", "MinionList", &NodeList{})
 }
 
 func (*Pod) IsAnAPIObject()                       {}

--- a/pkg/api/v1/conversion_test.go
+++ b/pkg/api/v1/conversion_test.go
@@ -23,29 +23,6 @@ import (
 	versioned "k8s.io/kubernetes/pkg/api/v1"
 )
 
-func TestNodeConversion(t *testing.T) {
-	obj, err := versioned.Codec.Decode([]byte(`{"kind":"Minion","apiVersion":"v1"}`))
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	if _, ok := obj.(*api.Node); !ok {
-		t.Errorf("unexpected type: %#v", obj)
-	}
-
-	obj, err = versioned.Codec.Decode([]byte(`{"kind":"MinionList","apiVersion":"v1"}`))
-	if err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-	if _, ok := obj.(*api.NodeList); !ok {
-		t.Errorf("unexpected type: %#v", obj)
-	}
-
-	obj = &api.Node{}
-	if err := versioned.Codec.DecodeInto([]byte(`{"kind":"Minion","apiVersion":"v1"}`), obj); err != nil {
-		t.Fatalf("unexpected error: %v", err)
-	}
-}
-
 // TestPodSpecConversion tests that ServiceAccount is an alias for
 // ServiceAccountName.
 func TestPodSpecConversion(t *testing.T) {

--- a/pkg/api/v1/register.go
+++ b/pkg/api/v1/register.go
@@ -86,9 +86,6 @@ func addKnownTypes() {
 		&ThirdPartyResourceList{},
 		&ThirdPartyResourceData{},
 	)
-	// Legacy names are supported
-	api.Scheme.AddKnownTypeWithName("v1", "Minion", &Node{})
-	api.Scheme.AddKnownTypeWithName("v1", "MinionList", &NodeList{})
 }
 
 func (*Pod) IsAnAPIObject()                       {}

--- a/pkg/kubectl/kubectl.go
+++ b/pkg/kubectl/kubectl.go
@@ -18,7 +18,6 @@ limitations under the License.
 package kubectl
 
 import (
-	"fmt"
 	"strings"
 
 	"k8s.io/kubernetes/pkg/api"
@@ -80,10 +79,6 @@ type ShortcutExpander struct {
 func (e ShortcutExpander) VersionAndKindForResource(resource string) (defaultVersion, kind string, err error) {
 	resource = expandResourceShortcut(resource)
 	defaultVersion, kind, err = e.RESTMapper.VersionAndKindForResource(resource)
-	// TODO: remove this once v1beta1 and v1beta2 are deprecated
-	if err == nil && kind == "Minion" {
-		err = fmt.Errorf("Alias minion(s) is deprecated. Use node(s) instead")
-	}
 	return defaultVersion, kind, err
 }
 

--- a/pkg/kubectl/resource/builder_test.go
+++ b/pkg/kubectl/resource/builder_test.go
@@ -580,7 +580,7 @@ func TestResourceTuple(t *testing.T) {
 			errFn: expectNoErr,
 		},
 		"valid multiple with namespaced and non-namespaced types": {
-			args:  []string{"minions/foo", "pod/bar"},
+			args:  []string{"nodes/foo", "pod/bar"},
 			errFn: expectNoErr,
 		},
 		"mixed arg types": {
@@ -621,7 +621,6 @@ func TestResourceTuple(t *testing.T) {
 					"/namespaces/test/pods/foo": runtime.EncodeOrDie(testapi.Default.Codec(), &pods.Items[0]),
 					"/namespaces/test/pods/bar": runtime.EncodeOrDie(testapi.Default.Codec(), &pods.Items[0]),
 					"/nodes/foo":                runtime.EncodeOrDie(testapi.Default.Codec(), &api.Node{ObjectMeta: api.ObjectMeta{Name: "foo"}}),
-					"/minions/foo":              runtime.EncodeOrDie(testapi.Default.Codec(), &api.Node{ObjectMeta: api.ObjectMeta{Name: "foo"}}),
 				}
 			}
 

